railties/test/plugin_helpers.rb

```diff
@@ -6,7 +6,7 @@ def generate_plugin(plugin_path, *args)
     prepare_plugin(plugin_path)
   end
 
-  def prepare_plugin(plugin_path)
+  def prepare_plugin(plugin_path, skip_bundler: true)
```

Comment:
Where does this argument come from?

Reply:
It was only added in case there were failures, but since tests passed I have removed it.